model/task_annotations/task_annotations.go

```diff
@@ -0,0 +1,30 @@
+package task_annotations
+
+import (
+	"time"
+)
+
+type TaskAnnotation struct {
+	Id string `bson:"_id" json:"id"`
+	// comment about the failure
+	Note string `bson:"note,omitempty" json:"note,omitempty"`
+	// links to tickets definitely related.
+	Issues []IssueLink `bson:"issues,omitempty" json:"issue,omitempty"`
+	// links to tickets possibly related
+	SuspectIssues []IssueLink `bson:"suspected_issues,omitempty" json:"suspected_issues,omitempty"`
+	// annotation attribution
+	Source AnnotationSource `bson:"source,omitempty" json:"source,omitempty"`
```

Comment:
I [hit this recently](https://github.com/evergreen-ci/evergreen/pull/4093): an embedded struct isn't really omitempty for bson or json.
Though it's not of great consequence...

Reply:
thanks!